### PR TITLE
Create initial Learning Sui Tokenomics overview

### PR DESCRIPTION
**Pull request by Clay-Mysten**
Adding the image in a separate PR
Will split these contents as specified once in

### CODE REVIEW

File: doc/src/learn/tokenomics/index.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+GasFees[t] = ComputationUnits[t] * ComputationPrice[t] + StorageUnits[t]*StoragePrice
+
+The gas functions ComputationUnits[t] and StorageUnits[t] measure the amount of computation and storage resources, respectively, required to process and store the data associated with t. The gas prices ConsumptionPrice[t] and StoragePrice translate the cost of computation and storage, respectively, into SUI units. The coupling between gas units and gas prices is useful since SUI’s market price will fluctuate over time depending on demand and supply fluctuations.
+
+## Computation gas prices
+
+The computation gas price ComputationPrice[t] captures the cost of one unit of computation in SUI units. This price is set at the transaction level and submitted by the user in two parts:
+
+ComputationPrice[t] = ReferencePrice + Tip[t], with Tip[t]>-ReferencePrice
```
Comment (huitseeker) [inline doc/src/learn/tokenomics/index.md:60]: I've pinged @econmysten elsewhere, but this one has me scratching my head.

**Review by huitseeker — COMMENTED** [review]
This would probably look a lot better by having proper typesetting, but that would probably be a revision by @econmysten. I'll let him close on the review, because I know he wanted links thrown in.

File: doc/src/learn/tokenomics/index.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    1. The total amount of stake rewards is distributed across various entities. Importantly, remember that the storage fund is taken into account in the calculation of the epoch’s total stake. However, the storage fund is not owned by any entities in the way that delegated SUI is. Instead, Sui’s economic model distributes the stake rewards accruing to the storage fund  – a share (1-alpha)% of the total stake rewards – to validators in order to compensate them for their storage costs. Of these rewards, a share gamma% is paid out to validators while the remaining (1-gamma)% is used to reinvest in the fund’s capital. Finally, assume that validators charge a commission delta% on SUI token holders as a fee for delegation. The split of stake rewards across participants is given by: 
+
+    	DelegatorRewards = (1-delta)*alpha*StakeRewards
+
+    	ValidatorRewards = (delta*alpha + gamma*(1-alpha))*StakeRewards
+
+    	Reinvestment = (1-gamma)*(1-alpha)*StakeRewards
```
[the remark this answers is not in the document]
Comment (econmysten) [reply doc/src/learn/tokenomics/index.md]: @huitseeker have updated with typesetting, thanks!

**Review by econmysten — APPROVED** [review]

**Review by huitseeker — COMMENTED** [review]
@econmysten Thanks for the typesetting update. Please review the typesetting of greeks: gamm and alpha in particular seems to not have gone through

**Pull request merged**